packages/flutter/test/material/dialog_test.dart

```diff
@@ -332,4 +334,47 @@ void main() {
       new Rect.fromLTRB(40.0, 24.0, 800.0 - 40.0, 600.0 - 24.0),
     );
   });
+
+  testWidgets('Dialog widget contains route semantics', (WidgetTester tester) async {
```

Comment:
Maybe add a test that ensures this when on iOS since this is one of the few cases where iOS also announces?

Reply:
I used a loop...hope that is okay for tests!